src/main/java/parser/DateTimeExtractor.java

```diff
@@ -31,6 +32,10 @@ public static LocalDateTime extractDateTime(String dateTimeFromUser, String comm
             dateEvent = LocalDateTime.parse(dateTimeFromUser, DATE_FORMATTER);
             return dateEvent;
         }
+        else if(command.equals("todo")) {
```

Comment:
Last note @AY1920S1-CS2113-T13-3/developers , is it important for  DateTimeExtractor to care what kind of command is calling it? Its purpose is to output a standardised date no matter what calls it, so it won't need multiple date variable and the if-else chain can be condensed to return one kind of date. The date error would be left as is